packages/react/src/components/button/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React, {ButtonHTMLAttributes} from 'react'
+import Spinner from '../spinner'
+
+interface ButtonOptions {
+  /**
+   * Default is `primary`. Style using `[data-sr-button='secondary']`.
+   * @type 'primary' | 'secondary' | 'tertiary'
+   */
+  kind?: 'primary' | 'secondary' | 'tertiary'
```

Comment:
`kind` is a frequent alternative to `type` which is often reserved html attribute.

Reply:
I'm giving this the side-eye because it deals with style and presentation not behavior, accessibility, and semantics which the rest of this does a great job of sticking to. I feel like `kind` is more likely a layer above button and a concern of the downstream consumer that will theme/style the button to suit their app's needs.

Reply:
removed! and since we're passing rest props you could still target it similarly down from `<Button>` component itself which is great imo. 👍